services/speechsynthesis.md

```diff
@@ -0,0 +1,43 @@
+#  Speech synthesis
+
+     identifier: 0x1204d995
+
+A speech synthesizer
+
+## Registers
+
+    ro speaking: bool @ reading
+
+Reports if the engine is speaking.
+
+    rw enabled: bool @ intensity
+
+Determines if the speech engine is in a non-paused state.
+
+    rw lang?: string @ 0x80
+
+Default language used for utterances.
```

Comment:
We should say how to specify it, I would say https://www.ietf.org/rfc/bcp/bcp47.txt (which is what HTML and XML use)

We could also rename this to `locale` (eg. en-us vs en-gb)

I would drop "Default", as I see no other way to change it